src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -2109,7 +2109,7 @@ internal enum ErrorCode
         ERR_FeatureNotAvailableInVersion11 = 9058,
         ERR_RefFieldInNonRefStruct = 9059,
         ERR_CannotMatchOnINumberBase = 9060,
-        ERR_MisplacedScoped = 9061,
+        // Available 9061,
```

Comment:
We shipped this error in 17.4-preview1. Is it fine to reuse it at this point?

Reply:
> We shipped this error in 17.4-preview1. Is it fine to reuse it at this point?

I think everything until RTM is a fair game